src/repositories/bookmarks.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { BookmarkModel } from "../models/Bookmark.model";
+
+export const listBookmarks = async () => {
+  const result = await BookmarkModel.find();
+  return result;
+};
+
+export const addOne = async (tweetId: string, ownerId: string) => {
```

Comment:
```suggestion
export const addOne =  (tweetId: string, ownerId: string) => {
```
await absent - async is not needed